suite/tests/CMakeLists.txt

```diff
@@ -2689,6 +2689,10 @@ endif ()
     use_DynamoRIO_extension(client.nudge_ex.dll drmgr)
   endif (X86)
 
+  if (X86)
```

Comment:
There's nothing x86-specific here is there?

Reply:
Done.